Kudu.Core/Environment.cs

```diff
@@ -101,6 +104,7 @@ public class Environment : IEnvironment
             _dataPath = Path.Combine(rootPath, Constants.DataPath);
             _jobsDataPath = Path.Combine(_dataPath, Constants.JobsPath);
             _jobsBinariesPath = Path.Combine(_webRootPath, Constants.AppDataPath, Constants.JobsPath);
+            _alwaysOnEnabled = IsAlwaysOnEnabled();
```

Comment:
I don't think this is the right place to put it, this class is about the kudu environment.
you can just keep it right where you need it.